tests/cli/workflow/mip/test_cli_mip_dna_run.py

```diff
@@ -5,26 +5,27 @@
 
 from cg.cli.workflow.mip_dna.base import run
 
-CASE_ID = "yellowhog"
-CONDA_ENV = "S_mip_rd-dna"
-EMAIL = "james.holden@scilifelab.se"
 
-
-def test_cg_dry_run(cli_runner, mip_context, caplog):
+def test_cg_dry_run(cli_runner, mip_context, caplog, case_id, email_adress):
```

Comment:
Maybe we could name the case_id fixture more descriptive? Such as valid_case_id or similar

Reply:
I think it is better to have the "valid" fixture name as it would be in the actual code i.e. "case_id" because that should be the main purpose of the fixture. If you want to use a faulty case_id, you can just replace it in the test or create a "faulty_case_id" fixture. That way the assumption is always a usable fixture unless stated otherwise. However, if this is not the the way cg fixtures should work I have no problem renaming it as you suggest. What do you say?

Reply:
I also think that `case_id` is informative enough, probably because I suggested it 😄

Reply:
Let us stick with that for now